document/src/main/java/com/ritense/document/web/rest/impl/JsonSchemaDocumentResource.java

```diff
@@ -138,6 +142,24 @@ public ResponseEntity<Void> assignHandlerToDocument(
         }
     }
 
+    @Override
+    @PostMapping(value = "/{documentId}/unassign")
+    public ResponseEntity<Void> unassignHandlerFromDocument(@PathVariable(name = "documentId")UUID documentId) {
+        logger.debug(String.format("REST call /api/document/%s/unassign", documentId));
+
+        try {
+            if (!hasAccessToDocumentId(documentId)) {
+                return ResponseEntity.badRequest().build();
+            }
+
+            documentService.unassignUserFromDocument(documentId);
+            return ResponseEntity.ok().build();
+        } catch (Exception e) {
+            logger.error("Failed to assign a user to a document", e);
+            return ResponseEntity.badRequest().build();
```

Comment:
Do we want to say bad request on any error? That's usually for wrong user input, right?